### DISCUSSION TITLE
Unstable audio speed and latency on ChromeOS

### DISCUSSION THREAD

**Discussion by BalazsSzeles** [ppy/osu#25802] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I was trying out the AppImage version of osu! on ChromeOS's Linux development environment. When it was starting, I noticed that the audio was faster than usual. Then it was fluctuating between it being too fast or too slow, and generally not stable (e. g. not being in sync with the osu! logo's metronome sound). The sound also had this crunchyness, which was especially noticeable on full screen dialogs. I also noticed that all this was happening with an unstable (but relatively good) FPS and latency. The game didn't felt bad in terms of raw performance. Also, the hardware isn't a problem, because the game ran flawlessly on an Ubuntu-based Linux.

### Screenshots or videos

[Here's a video where I recreated the steps of the issue.](https://github.com/ppy/osu/assets/71432767/9a2a63ac-6475-4106-a171-9359fea0049e) I don't feel like that the recording really impacted the performance of the game that much. The main focus is on the audio and the FPS counter.

### Version

2023.1130.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/13696601/runtime.log)
[performance.log](https://github.com/ppy/osu/files/13696602/performance.log)
[network.log](https://github.com/ppy/osu/files/13696603/network.log)
[input.log](https://github.com/ppy/osu/files/13696604/input.log)
[database.log](https://github.com/ppy/osu/files/13696605/database.log)

**Comment by Hexality** [comment]
I think this belongs on the [issues](https://github.com/ppy/osu/issues) section...

**Comment by insecurestudios** [comment]
While this does not solve your problem, I would suggest using either the [flatpak version](https://flathub.org/apps/sh.ppy.osu) or android for better results.

**Comment by sineplusx** [comment, marked as answer]
A fix for this issue has been found, see [#28366](https://github.com/ppy/osu/discussions/28366#discussioncomment-9624201).
